app/models/post.rb

```diff
@@ -0,0 +1,17 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :likes
+  has_many :comments
+
+  after_save :update_posts_count
+
+  def recent_five_comments
+    comments.order(create_at: :desc).limit(5)
+  end
+
+  private
+
+  def update_posts_count
+    author.increment(:post_counter)
```

Comment:
- You have actually done well so far, but there is just a minor issue to be fixed so your project does not have bugs in the future, the increment method is missing the `!`. Please fix this 🙏. It should be ```author.increment!(:post_counter)```

Reply:
@mckent05  Thanks for the feedback. Appreciate it.

Reply:
resolved

Reply:
- Well resolved 👍🏽